app/views/devise/sessions/_form.html.haml

```diff
@@ -0,0 +1,12 @@
+- form_url = params[:oauth] == 'true' ? person_session_path(oauth: true) : person_session_path
```

Comment:
gibt es einen Grund _form zu überschreiben und nicht aus dem core zu übernehmen?
einziger Unterschied den ich sehe ist Zeile 10/12 sind vertauscht. 
falls die Reihenfolge hier keine Rolle spielt würde ich auf jedenfall das file aus dem core übernehmen.

dies hätte auch den Vorteil das wir überall die gleiche Login form haben. Das Design wird dann im Wagon einfach über die übergeordnete 'new.html.haml' gesteuert.

falls das nicht geht, lizenz header noch ergänzen in diesem File

Reply:
Nein mitlerweile gibt es keinen Grund mehr. Merci fürs aufpassen.

Anfangs hatte ich hier noch mehr angepasst aber mich dann dagegen entschieden und es quasi wieder zum core Stand reverted und dabei vergessen das File einfach rauszunehmen :)